pkg/doc/bbs/bbs12381g2pub/internal/g1.go

```diff
@@ -0,0 +1,688 @@
+/*
+Taken from https://github.com/kilic/bls12-381
+Changes:
+1) Use black2b as hash function for HashToCurve().
```

Comment:
Please move as many changes as possible to a new file.

Reply:
Created `g1_custom.go`.